deployment/template.env

```diff
@@ -1,48 +1,49 @@
-# Set variables below and copy to .env
-#
-# NOTE: ALL VARIABLES MUST BE SET. IF NOT SET, PLEASE COMMENT IT OUT.
-# IF COMMENTED OUT, DEFAULT VALUES WILL BE USED FROM `core_backend/app/config/app_config.py
+# IMPORTANT: all variables must be either set or commented out.
+# if commented out, default values will be used from `core_backend/app/config/app_config.py`
 
+#### Root domain (if not set, localhost will be used)
+# DOMAIN=
```

Comment:
Should we add something like this? (could get confused about full URL vs just domain (I did !)
```suggestion
# DOMAIN=example.domain.com
```